tests/Unit/ApparentHorizons/StrahlkorperGrTestHelpers.cpp

```diff
@@ -34,6 +34,29 @@ tnsr::ii<DataType, SpatialDim, Frame> make_spatial_ricci_schwarzschild(
   return ricci;
 }
 
+template <size_t SpatialDim, typename Frame, typename DataType>
```

Comment:
just so you are aware, we are working on a way of getting rid of all these `make_*` functions because they're annoying, cause code bloat, and introduce endless magic numbers into the code.